MusicTaste/MusicTaste/View Controllers/Login/LoginViewController.m

```diff
@@ -1,98 +1,72 @@
+            [self performSegueWithIdentifier:@"tabBarSegue" sender:nil];
+        }
+        else{
+            //If the username and password are not in the data base return Invalid Login UIAlert
+            UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Invalid Login"
+                                                                           message:@"Username and/or password incorrect."
+                                                                           preferredStyle:(UIAlertControllerStyleAlert)];
+            UIAlertAction *okAction = [UIAlertAction actionWithTitle:@"OK"
+                                                               style:UIAlertActionStyleDefault
+                                                             handler:^(UIAlertAction * _Nonnull action) {
+                                                             }];
+            [alert addAction:okAction];
+            [self presentViewController:alert animated:YES completion:^{
+            }];
+        }
     }];
 }
-
-#pragma mark - SPTSessionManagerDelegate
-
-- (void)sessionManager:(SPTSessionManager *)manager didInitiateSession:(SPTSession *)session
-{
-    dispatch_async(dispatch_get_main_queue(), ^{
-        [self presentAlertControllerWithTitle:@"Authorization Succeeded"
-                                      message:session.description
-                                  buttonTitle:@"Nice"];
-        [self performSegueWithIdentifier:@"loginSegue" sender:nil];
-        
-    });
-}
-
-
-
-- (void)sessionManager:(SPTSessionManager *)manager didFailWithError:(NSError *)error
-{
-    [self presentAlertControllerWithTitle:@"Authorization Failed"
-                                  message:error.description
-                              buttonTitle:@"Bummer"];
-}
-
-- (void)sessionManager:(SPTSessionManager *)manager didRenewSession:(SPTSession *)session
-{
-    [self presentAlertControllerWithTitle:@"Session Renewed"
-                                  message:session.description
-                              buttonTitle:@"Sweet"];
-}
-
-#pragma mark - Set up view
-
-- (void)loadView
-{
-    ConnectView *view = [ConnectView new];
-    [view.connectButton addTarget:self action:@selector(didTapAuthButton:) forControlEvents:UIControlEventTouchUpInside];
-    self.view = view;
+- (IBAction)signUp:(id)sender {
+    //If the user signs up the password and username will be added to the database
+    PFUser *newUser = [PFUser user];
+    newUser.username = self.usernameField.text;
+    newUser.password = self.passwordField.text;
+    [newUser signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * error) {
+        if (error != nil) {
+            NSLog(@"Error: %@", error.localizedDescription);
+        } else {
+            NSLog(@"User registered successfully");
+        }
+    }];
 }
 
-- (void)presentAlertControllerWithTitle:(NSString *)title
```

Comment:
This function is unnecessary for now because I pivoted to transitioning to LoginViewController; however, I might implement this later.

Reply:
Looks like the function was moved to ConnectViewController